plexpy/pmsconnect.py

```diff
@@ -1896,51 +1896,24 @@ def get_image(self, img=None, width=None, height=None, fallback=None):
                                 height { the image height }
         Output: array
         """
-        if not img:
-            logger.error(u"PlexPy Pmsconnect :: Image proxy queried but no input received.")
-            return None
-
-        # Remove the timestamp from PMS image uri
-        image_uri = img.rsplit('/', 1)[0]
-
-        # Try to retrieve the image from cache if it isn't a bif index.
-        if not 'indexes' in image_uri:
-            image_path, content_type = helpers.cache_image(image_uri)
 
-            if image_path and content_type:
-                return [open(image_path, 'rb'), content_type]
-
-        try:
+        if img:
+            uri = '/photo/:/transcode?url=http://127.0.0.1:32400%s' % img
             if width.isdigit() and height.isdigit():
```

Comment:
I think the uri _must_ include a with and height. I get `404 Bad Request` if I don't include the dimensions.